internal/yggdrasil/yggdrasil.go

```diff
@@ -142,6 +142,7 @@ func (h *Handler) getNamespace(ctx context.Context) string {
 func (h *Handler) GetControlMessageForDevice(ctx context.Context, params yggdrasil.GetControlMessageForDeviceParams) middleware.Responder {
 	deviceID := params.DeviceID
 	if !IsOwnDevice(ctx, deviceID) {
+		h.metrics.IncEdgeDeviceInvalidOwnerCounter()
 		return operations.NewGetControlMessageForDeviceForbidden()
 	}
```

Comment:
this block repeats itself 4 times.
but this is one of that cases in which refactoring into a function will not save LOC, but will allow to encapsulate:
* Owner validation
* Increase metrics counter
* Report the same error for this violation

however, I'm not strongly pushing to it. just an idea. going to give my LGTM regardless.